package.json

```
@@ -1,18 +1,26 @@
 {
   "name": "lighthouses",
-  "version": "0.0.1",
+  "version": "0.0.2",
   "description": "",
-  "main": "index.js",
+  "main": "index.ts",
   "type": "module",
+  "packageManager": "pnpm@8.13.1",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
-  },
-  "author": "",
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "type-check": "tsc --noEmit"
+  },  
+  "collaborators": ["Charlie Kornoelje (https://www.github.com/charkour", "Duncan Van Keulen (https://www.github.com/TheDunco)"],
```

I think this is missing a closing `)`
```suggestion
  "collaborators": ["Charlie Kornoelje (https://www.github.com/charkour)", "Duncan Van Keulen (https://www.github.com/TheDunco)"],
```

---

Ah, indeed, good catch!